frontend/i18n/en.json

```
@@ -314,6 +314,31 @@
       "webhook": "Webhook",
       "networks": "Networks"
     },
+    "general": {
+      "do_not_disturb": "Do not disturb",
+      "mute": {
+        "mutes_all": "Mutes all notifications",
+        "unmute": "Unmute",
+        "muted_until_date": "Muted until {date}",
+        "muted_until_turned_on": "Muted until turned back on",
+        "select_duration": "Select mute duration",
+        "count_hours": "{count} hour|{count} hours",
+        "until_turned_on": "Until I turn it back on"
```

nitpick: (Imo a little more consistent, but naming things is hard)
```suggestion
      "mute": {
        "all": "Mutes all notifications",
        "unmute": "Unmute",
        "until": "Muted until {date}",
        "until_turned_on": "Muted until turned back on",
        "select_duration": "Select mute duration",
        "hours": "{count} hour|{count} hours",
        "until_turned_on": "Until I turn it back on"
```